config/orm.php

```diff
@@ -0,0 +1,367 @@
+
+        ->set('doctrine.orm.entity_manager.abstract', param('doctrine.orm.entity_manager.class'))
+            ->abstract()
+            ->lazy()
+
+        ->set('doctrine.orm.container_repository_factory', ContainerRepositoryFactory::class)
+            ->args([
+                inline_service(ServiceLocator::class)->args([
+                    [],
+                ]),
+            ])
+
+        ->set('doctrine.orm.manager_configurator.abstract', param('doctrine.orm.manager_configurator.class'))
+            ->abstract()
+            ->args([
+                [],
+                [],
+            ])
+
+        ->set('doctrine.orm.validator.unique', param('doctrine.orm.validator.unique.class'))
+            ->tag('validator.constraint_validator', ['alias' => 'doctrine.orm.validator.unique'])
+            ->args([
+                service('doctrine'),
+            ])
+
+        ->set('doctrine.orm.validator_initializer', param('doctrine.orm.validator_initializer.class'))
+            ->tag('validator.initializer')
+            ->args([
+                service('doctrine'),
+            ])
+
+        ->set('doctrine.orm.security.user.provider', param('doctrine.orm.security.user.provider.class'))
+            ->abstract()
+            ->args([
+                service('doctrine'),
+            ])
+
+        ->set('doctrine.orm.listeners.resolve_target_entity', param('doctrine.orm.listeners.resolve_target_entity.class'))
+
+        ->set('doctrine.orm.listeners.doctrine_dbal_cache_adapter_schema_listener', DoctrineDbalCacheAdapterSchemaListener::class)
+            ->args([
+                [],
+            ])
+            ->tag('doctrine.event_listener', ['event' => 'postGenerateSchema'])
+
+        ->set('doctrine.orm.listeners.doctrine_token_provider_schema_listener', RememberMeTokenProviderDoctrineSchemaListener::class)
+            ->args([
+                tagged_iterator('security.remember_me_handler'),
+            ])
+            ->tag('doctrine.event_listener', ['event' => 'postGenerateSchema'])
+
+        ->set('doctrine.orm.listeners.pdo_session_handler_schema_listener', PdoSessionHandlerSchemaListener::class)
+            ->args([
+                service('session.handler'),
+            ])
+            ->tag('doctrine.event_listener', ['event' => 'postGenerateSchema'])
+
+        ->set('doctrine.orm.listeners.lock_store_schema_listener', LockStoreSchemaListener::class)
+            ->args([
+                tagged_iterator('lock.store'),
+            ])
+            ->tag('doctrine.event_listener', ['event' => 'postGenerateSchema'])
+
+        ->set('doctrine.orm.naming_strategy.default', param('doctrine.orm.naming_strategy.default.class'))
+
+        ->set('doctrine.orm.naming_strategy.underscore', param('doctrine.orm.naming_strategy.underscore.class'))
+
+        ->set('doctrine.orm.naming_strategy.underscore_number_aware', param('doctrine.orm.naming_strategy.underscore.class'))
+            ->args([
+                'CASE_LOWER',
```

Comment:
@nicolas-grekas this should be an integer (the PHP constant), according to the UnderscoreNamingStrategy class

Just noticed in some deprecations :)

Reply:
Done, already merged by @stof :)